dist/SocialCalc.js

```diff
@@ -25090,6 +25064,8 @@ SocialCalc.SpreadsheetControlExecuteCommand = function(obj, combostr, sstr) {
    var spreadsheet = SocialCalc.GetSpreadsheetControlObject();
    var eobj = spreadsheet.editor;
 
+   console.log("SCEC : combostr : " + combostr + "\n\tsstr : " + sstr);             // DEBUG STUFF
```

Comment:
These probably should be removed

Reply:
Yes, I agree. How should I do this ? Push a new commit on top of this one or amend this one and force push or something else ?